core/backend/src/SchemaSync.ts

```diff
@@ -0,0 +1,55 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+/** @packageDocumentation
+ * @module SQLiteDb
+ */
+
+import { CloudSqlite } from "./CloudSqlite";
+import { VersionedSqliteDb } from "./SQLiteDb";
+
+
+/** @beta */
+export namespace SchemaSync {
+
+  /**
+   * A SQLite database for storing schemas.
+   */
+  export class SchemaSyncDb extends VersionedSqliteDb {
+    public override readonly myVersion = "1.0.0";
+    protected override createDDL() { }
+  }
+
+  const defaultDbName = "SharedSchemaChannelDb" as const;
```

Comment:
I don't understand why this is called "shared schema channel". We use the term "channel" for access control to element and this has nothing to do with that. Also the term "shared" is confusing. This is about synchronizing schema changes.  I think the terminology for the whole concept should refer to "synchronizing schemas"

Reply:
Fixed. All Shared/Channel -> Schema Sync